TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithAlterSequence.rdzSQL.txt

```diff
@@ -0,0 +1,366 @@
+﻿--- Diagnostics ---
+Line 16[21,25] <27, Error, Syntax> - Syntax error : In RESTART value, only digits '0' are allowed after decimal point RuleStack=codeElement>alterSequenceStatement,  OffendingSymbol=[21,25:ALTER]<SQL_ALTER>
```

Comment:
RuleStack can be removed from these diagnostics

Reply:
The RuleStack does not appear in the message displayed in IDz. Besides, we already use this overload of `DiagnosticUtils.AddError` in other CodeElement checkers.